modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require 'claims_api/v2/error/lighthouse_error_handler'
+require 'claims_api/v2/json_format_validation'
+
+module ClaimsApi
+  module V2
+    module Veterans
+      class PowerOfAttorney::RequestController < ClaimsApi::V2::Veterans::PowerOfAttorney::BaseController
+        FORM_NUMBER = 'POA_REQUEST'
+
+        def request_representative
+          target_veteran
+          @claims_api_forms_validation_errors = validate_form_2122_and_2122a_submission_values(user_profile)
+          validate_json_schema(FORM_NUMBER)
+          validate_accredited_representative(form_attributes.dig('poa', 'registrationNumber'),
+                                             form_attributes.dig('poa', 'poaCode'))
+          validate_accredited_organization(form_attributes.dig('poa', 'poaCode'))
+
+          # if we get here, the only errors not raised are form value validation errors
+          if @claims_api_forms_validation_errors
+            raise ::ClaimsApi::Common::Exceptions::Lighthouse::JsonDisabilityCompensationValidationError,
```

Comment:
We should clean this error class name up since it is getting used outside of Disability Compensation